columnLayout.js

```diff
@@ -43,10 +41,16 @@ var countBlock = function(){
 		start = middleWidth - start;
 	}
 
+	// This default value sets top space from where this block starts $('.navbar') is the selecter to select top element
+	var topEle = parseInt($('.navbar').css('height'), 10) + parseInt($('.wrapper').css('margin-top'), 10);
```

Comment:
Make this change into quote saga / columnlayout.js. Keep this as a general so that any one can use it